src/ansi-c/c_typecheck_expr.cpp

```diff
@@ -44,6 +44,94 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <sstream>
 
+/// Architecturally similar to \ref can_forward_propagatet, but specialized for
+/// what is a constexpr, i.e., an expression that can be fully evaluated at
+/// compile time.
+class is_compile_time_constantt
```

Comment:
May I suggest explicitly referencing the standard, and where the logic deviates from the standard to mimic gcc or clang.

Reply:
This is now done.